yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -216,6 +242,41 @@ module vyatta-protocols-frr-bgp-v1 {
 		}
 	}
 
+  grouping network-ipv4-vpn {
+    list network {
+      description "BGP network";
+      configd:help "BGP network";
+      key "addr";
+      leaf addr {
+        description "BGP network";
+        type types:ipv4-prefix {
+          configd:normalize "normalize ipv4-prefix";
```

Comment:
why is this required from ? Doesn't it make more sense to specify the network from the routing instance for the address family  ? Which rd is used if you configure e.g. 

set protocols bgp 65201 address-family ipv4-vpn network 1.2.3.0/24